scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/job/InternalTaskFlowJob.java

```diff
@@ -37,6 +37,7 @@
 package org.ow2.proactive.scheduler.job;
 
 import java.util.ArrayList;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.ArrayList'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)